source/conf.py

```diff
@@ -344,3 +344,37 @@
 # If true, figures, tables and code-blocks are automatically numbered if they
 # have a caption. At same time, the numref role is enabled.
 numfig = True
+
+# -- Custom configuration -------------------------------------------------
+
+from jinja2 import Environment, StrictUndefined
+
+# Use Jinja to parse RST files first
+def rstjinja(app, docname, source):
+    """
+    Render our pages as a jinja template for fancy templating goodness.
+    """
+    jinja_env = Environment(undefined=StrictUndefined)
```

Comment:
I'll ignore this for now and merge. But if we start documenting anything here that would require the documentation to be escaped for jinja delimiters I'm going to change all of them then to use three `{{{` prefixed and `}}}` suffixed.